sqlup-mode.el

```diff
@@ -67,19 +67,12 @@
 this mode's logic will be evaluated.")
 
 (defconst sqlup-eval-keywords
-  '((postgres "EXECUTE"))
-"List of keywords introducing eval strings, organised by dialect.")
+  '((postgres "EXECUTE" "format("))
```

Comment:
I'm not sure what you mean. We should have the option of having either EXECUTE only or EXECUTE format( preceding our string. The alist should contain both forms, and we should try both with looking-back. Did I botch the PR ?

I'm currently on holiday for the next few days, but I'll try to sort this out.